src/main/java/helperInstrument/EnviromentHelper.java

```diff
@@ -31,6 +31,11 @@ public static String getMasterPass () {
 		return masterPass;
 	}
 	
+	public static String getAlternativePass(String alternativePass){
+		masterPass = alternativePass;
```

Comment:
why are you setting something in get method ???

Reply:
actualy i don`t need it i will remove it